#int32[]  robot_pos_x                                      
#int32[]  robot_pos_y                               
#int32[]  task_pos_x
#int32[]  task_pos_y

int32 team_number
int32 task_number
bool  finish_episode
bool  finish_train
